droidlet/lowlevel/locobot/locobot_mover_utils.py

```diff
@@ -110,3 +135,60 @@ def xyz_pyrobot_to_canonical_coords(xyz):
 def xyz_canonical_coords_to_pyrobot_coords(xyz):
     """converts 3D coords from canonical to pyrobot coords."""
     return xyz @ np.linalg.inv(pyrobot_to_canonical_frame)
+
+
+class ExaminedMap:
```

Comment:
put this in a separate file in droidlet.memory.robot, and import in the loco_memory...